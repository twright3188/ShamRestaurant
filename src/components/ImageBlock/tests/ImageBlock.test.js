import React from 'react';
import renderer from 'react-test-renderer';
import { DeliveryImageBlock } from '../index';

it ('image block renders correctly', () => {
    const tree = renderer.create(<DeliveryImageBlock image='image' title='title' result={40} />).toJSON();
    expect(tree).toMatchSnapshot();
});