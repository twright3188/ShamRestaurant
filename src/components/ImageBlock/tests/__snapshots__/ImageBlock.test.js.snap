// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`image block renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "borderBottomColor": "rgba(34, 38, 36, 0.15)",
      "borderBottomWidth": 1,
      "flexDirection": "row",
      "paddingHorizontal": 15,
      "paddingVertical": 16,
      "width": "100%",
    }
  }
>
  <Image
    source={
      Object {
        "uri": "image",
      }
    }
    style={
      Object {
        "borderRadius": 40,
        "height": 80,
        "marginRight": 16,
        "width": 80,
      }
    }
  />
  <View
    style={
      Object {
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "flexWrap": "wrap",
            "fontFamily": "Helvetica",
            "fontWeight": "400",
          },
          Object {
            "color": "#222624",
            "fontSize": 16,
            "lineHeight": 24,
          },
          Object {
            "color": "#78807B",
            "fontWeight": "bold",
          },
        ]
      }
    >
      title
    </Text>
    <Text
      style={
        Array [
          Object {
            "flexWrap": "wrap",
            "fontFamily": "Helvetica",
            "fontWeight": "400",
          },
          Object {
            "color": "#222624",
            "fontSize": 16,
            "lineHeight": 24,
          },
          Object {
            "fontWeight": "bold",
          },
        ]
      }
    >
      40
    </Text>
  </View>
</View>
`;
